lua/darkrp_modules/levels/sh_config.lua

```diff
@@ -54,98 +67,25 @@ Printer.XPPerPrint = 10 // How much XP to give a player every time they print.
 Printer.MoneyPerPrint = 50 // How much money to give a player every time they print.
 Printer.Color = Color(255,255,255,255) // The color of the printer. Setting it to (255,255,255,255) will make it the normal prop color.
 Printer.Model = 'models/props_lab/reciever01b.mdl' // The model of the printer. To find the path of a model, right click it in the spawn menu and click "Copy to Clipboard"
-Printer.Prestige = 0 // The prestige you have to be to buy the printer. Only works with the prestige DLC.
+Printer.Prestige = 0 // The prestige you have to be to buy the printer. Only works with the prestige DLC on Gmodstore.
 Printer.Allowed = {} // Same as DarkRP .allowed
 table.insert(Printers,Printer) // Leave this line
 */
 
 // Default printers:
+// they have been deleted since they were not working, they will be back in a few updates
+
+// Default xp books:
+// they have been deleted since they were not working, they will be back in a few updates
```

Comment:
If you have fixed books then feel free to add them, if you haven't then remove everything to do with them for the merge

Reply:
i fixed them right now :) they now works like printer config